addons/binding/org.openhab.binding.rfxcom/src/main/java/org/openhab/binding/rfxcom/handler/RFXComBridgeHandler.java

```diff
@@ -232,10 +236,12 @@ public void packetReceived(byte[] packet) {
                 if (message instanceof RFXComInterfaceMessage) {
                     RFXComInterfaceMessage msg = (RFXComInterfaceMessage) message;
                     if (msg.subType == SubType.RESPONSE) {
-                        logger.debug("RFXCOM transceiver/receiver type: {}, hw version: {}.{}, fw version: {}",
+                        if (msg.command == Commands.GET_STATUS) {
+                            logger.info("RFXCOM transceiver/receiver type: {}, hw version: {}.{}, fw version: {}",
                                 msg.transceiverType, msg.hardwareVersion1, msg.hardwareVersion2, msg.firmwareVersion);
+                            getThing().setProperty(PROPERTY_HARDWARE_VERSION, msg.hardwareVersion1 + "." + msg.hardwareVersion2);
```

Comment:
Could you use the default properties for this https://github.com/eclipse/smarthome/blob/master/bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/Thing.java#L50